monai/losses/dice.py

```diff
@@ -701,12 +701,10 @@ def ce(self, input: torch.Tensor, target: torch.Tensor):
 
         """
         n_pred_ch, n_target_ch = input.shape[1], target.shape[1]
-        if n_pred_ch == n_target_ch:
```

Comment:
The changes break unit tests https://github.com/Project-MONAI/MONAI/actions/runs/3208441811/jobs/5244297528

Reply:
it fails on old pytorch versions, so it seems on Old pytorch version, CE didn't support multichannel input. 
do we need to support those old pytorch versions?